bootstrapper-maven-plugin/src/main/resources/templates/Spec.java

```diff
@@ -0,0 +1,15 @@
+package {{groupId}};
+
+public class {{artifactClassId}}Spec {
+
+    private String value;
```

Comment:
Not sure we should be adding a value here since it will always often be removed and changing the value name / type is more work than just adding one from scratch, imo.

Reply:
The idea is to have a minimalistic working project generated in these generators. So added a very simple implementation that can be instantly tried out.

Reply:
Yes, I understand the point but as a user, I'm annoyed when I get generated code that's useless. My opinion is that if it isn't code that the user can use as-is or with only minimal changes, then it shouldn't be generated. In this case, it's easier to add the fields directly instead of having to have one generated and having to then change it afterwards, imo.

Reply:
Agree with create a field here, it would help people to getting started. Although it is indeed a noise for advanced users.

Reply:
I'm thinking that it should be obvious that fields need to be added to the spec when creating a primary resource but if it's not, maybe then a comment should be generated to explain the purpose of this class and the status one. A comment would be better to help people get started since it would clarify the intent without creating noise that would interfere with the eventual functionality to be implemented.

Reply:
I guess we'll have to agree to disagree. Especially considering that people will need to remember to re-apply the CRD each time they modify their CR, this might cause confusion for users to have something generated that's wrong and then they might forget to update it when they actually modify it. So adding a dummy value here is not really helping anything because that's not really the "hard" part of getting the operator to run in the first place.